### ISSUE TITLE
Binary should be named _mocha to make the npm package a drop-in replacement

### ISSUE THREAD

**Issue by rulatir** [stefanpenner/mocha-esm#8]
The binary in the package should be named `_mocha` to make the npm package a drop-in replacement. Some test runners ask for the path to mocha package under node_modules, rather than for a mocha binary. They then use the standard Mocha package's convention to locate the binary. As a result, it is impossible to hook mocha-esm to such test runners. The PHPStorm Mocha plugin is one such test runner.

**Comment by stefanpenner** [comment]
Makes sense. I dont have time right now to confirm this all works, but it sound reasonable.
